index.html

```diff
@@ -70,12 +70,17 @@ <h3 id="advantages">Advantages of wheels</h3>
                 <h2 id="about-list">What is this list?</h2>
                 <p>This site shows the top 360 most-downloaded packages on <a href="https://pypi.org/">PyPI</a> showing which have been uploaded as wheel archives.</p>
                 <ul>
-                    <li><span class="text-success">Green</span> packages offer wheels,</li>
+                    <li><span class="text-success">Green</span> packages offer wheels compatible with the newest Python version (updated shortly before its final release),</li>
+                    <li><span class="text-warning">Orange</span> packages offer older wheels,</li>
                     <li><span class="text-muted">White</span> packages have no wheel archives uploaded (yet!).</li>
                 </ul>
+                <p>Currently, Python 3.13 is considered the newest Python version.</p>
```

Comment:
Thanks. Is it possible to use the NEWEST_PYTHON_VER value here? Otherwise I am afraid the version might get out of sync later.

Reply:
AFAIK, we'd have to generate index.html from a template rather than use the file verbatim as it is now. So I guess that would require pulling jinja2 and adding the generating machinery? Maybe there is an easier way?